cfme/tests/configure/test_access_control.py

```diff
@@ -2674,3 +2674,142 @@ def test_tenant_template_visibility():
         initialEstimate: 1/10h
     """
     pass
+
+
+@pytest.mark.manual
+@test_requirements.rbac
```

Comment:
same here for the requirements, I think it's better to define at the module level